config/runtime.exs

```diff
@@ -227,14 +210,33 @@ cond do
 end
 
 if(File.exists?("cacert.pem") && File.exists?("cert.pem") && File.exists?("cert.key")) do
-  ssl_opts = [
-    cacertfile: "cacert.pem",
-    certfile: "cert.pem",
-    keyfile: "cert.key",
-    verify: :verify_peer
-  ]
+  config :logflare,
+    ssl: [
+      cacertfile: "cacert.pem",
+      certfile: "cert.pem",
+      keyfile: "cert.key",
+      verify: :verify_peer
+    ]
+end
 
-  config :logflare, ssl: ssl_opts
+if(
+  System.get_env("DB_SSL") == "true" && File.exists?("db-server-ca.pem") &&
+    File.exists?("db-client-ca.pem") && File.exists?("db-client-key.pem")
```

Comment:
I am pretty sure that checking for paths like that can fail in release, as these files may not be where we want them to be. Maybe instead use `DB_SSL` as a path to where these files will be stored? That should make the configuration more flexible.

Reply:
agreed. yes we can update the certfile loading method for both db and grpc, can do this in a separate PR.